infrastructure/lib/atat-web-api-stack.ts

```diff
@@ -134,7 +136,30 @@ export class AtatWebApiStack extends cdk.Stack {
   }
 
   private addTaskOrderRoutes(props?: AtatWebApiStackProps) {
+    // Creates a target server access log bucket shared amongst the Task Order Lifecycle buckets
+    // server access logs enabled on target bucket
+    const taskOrdersAccessLogsBucket = new SecureBucket(this, "taskOrdersLogBucket", {
+      logTargetBucket: "self",
+      logTargetPrefix: "logs/logbucket/",
+      bucketProps: {
+        accessControl: s3.BucketAccessControl.LOG_DELIVERY_WRITE,
+      },
+    });
```

Comment:
We still need a more global logging target bucket imo rather than one big shared one. but maybe this is fine for now

Reply:
Left the way it is for now.